vkcs/kubernetes/resource_cluster.go

```diff
@@ -137,6 +137,7 @@ func ResourceKubernetesCluster() *schema.Resource {
 				Type:        schema.TypeString,
 				ForceNew:    true,
 				Optional:    true,
+				Deprecated:  "SSH connection to kubernetes clusters is no longer supported in vk cloud.",
```

Comment:
Is it? SSH connections are supported for sure, but maybe keypair is ignored.

Reply:
SSH connection to kubernetes clusters using a key pair is no longer supported in vk cloud
Is that right?

Reply:
I would omit info about SSH connection at all. Just something about specifying Nova key pairs. Btw, have you tried to specify it?